javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -233,7 +233,7 @@ Object.getPrototypeOf(Player.prototype) === Object.prototype; // true
 
 // Output may slightly differ based on the browser
 player1.valueOf(); // Output: Object { name: "steve", marker: "X", sayName: sayName() }
-~~~
+
```

Comment:
Looks like there was an issue from the merge conflicts. We need to add back in the tilde `~~~` here as well as your original update

Reply:
Just saw that. Thanks!

Reply:
Did you still want to update the `Output: Object` line to `Output: Player`?

Reply:
Either way is good
I think the comment that says that the output may differ is good enough

Reply:
In that case I will close this PR out